[Bug][DevOps] The campaigns module build fails on Apple M1 chip

As the issue title states, the `campaigns` build fails on Apple silicon.

**Steps to reproduce:**
1. `git clone git@github.com:daritelska-platforma/frontend.git && cd frontend`
2. `ln -s .env.example .env`
3.  `docker-compose build --no-cache campaigns`

**Expected result:**
The `campaigns` module is built successfully.

**Actual result:**
The `campaigns` module build fails.

**Environment:**
- OS: **macOS Big Sur**
- Chip: **Apple M1**
- Docker Desktop: **v3.3.3 (64333)**

**Logs:**
<details>
  <summary>Click to expand!</summary>

```
Building campaigns
[+] Building 30.6s (14/16)
 => [internal] load build definition from Dockerfile                                                                                                     0.0s
 => => transferring dockerfile: 835B                                                                                                                     0.0s
 => [internal] load .dockerignore                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                          0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:5.0                                                                                        0.3s
 => [internal] load metadata for mcr.microsoft.com/dotnet/aspnet:5.0                                                                                     0.3s
 => [base 1/2] FROM mcr.microsoft.com/dotnet/aspnet:5.0@sha256:41faa82e297e4067a5d6c8803e4dbb64709880047f42ae6671820dd3bfff6ad6                          0.0s
 => CACHED [build 1/6] FROM mcr.microsoft.com/dotnet/sdk:5.0@sha256:e6b7bbd8daa77b072c6f1e0c18f531e63381cc6c8d75ee922fa9fd504749966e                     0.0s
 => [internal] load build context                                                                                                                        0.0s
 => => transferring context: 5.92kB                                                                                                                      0.0s
 => CACHED [base 2/2] WORKDIR /app                                                                                                                       0.0s
 => CACHED [final 1/2] WORKDIR /app                                                                                                                      0.0s
 => [build 2/6] RUN apt-get update  && DEBIAN_FRONTEND=noninteractive     apt-get install --no-install-recommends --assume-yes       protobuf-compiler-  8.4s
 => [build 3/6] COPY module-campaigns/src/ module-campaigns/src/                                                                                         0.0s
 => [build 4/6] COPY protos/ protos/                                                                                                                     0.0s
 => [build 5/6] WORKDIR /module-campaigns/src/Podkrepibg.Campaigns                                                                                       0.0s
 => ERROR [build 6/6] RUN dotnet build "Podkrepibg.Campaigns.csproj" -c Release -o /app/build                                                           21.6s
------
 > [build 6/6] RUN dotnet build "Podkrepibg.Campaigns.csproj" -c Release -o /app/build:
#14 0.445 Microsoft (R) Build Engine version 16.10.0+4242f381a for .NET
#14 0.445 Copyright (C) Microsoft Corporation. All rights reserved.
#14 0.445
#14 0.725   Determining projects to restore...
#14 1.283   Restored /module-campaigns/src/Podkrepibg.Campaigns.Domain/Podkrepibg.Campaigns.Domain.csproj (in 57 ms).
#14 19.66   Restored /module-campaigns/src/Podkrepibg.Campaigns.Application/Podkrepibg.Campaigns.Application.csproj (in 18.43 sec).
#14 19.66   Restored /module-campaigns/src/Podkrepibg.Campaigns/Podkrepibg.Campaigns.csproj (in 18.43 sec).
#14 19.66   Restored /module-campaigns/src/Podkrepibg.Campaigns.Infrastructure/Podkrepibg.Campaigns.Infrastructure.csproj (in 18.43 sec).
#14 21.28   Podkrepibg.Campaigns.Domain -> /app/build/Podkrepibg.Campaigns.Domain.dll
#14 21.43 /root/.nuget/packages/grpc.tools/2.37.0/build/_protobuf/Google.Protobuf.Tools.targets(280,5): error MSB6006: "/root/.nuget/packages/grpc.tools/2.37.0/tools/linux_arm64/protoc" exited with code 139. [/module-campaigns/src/Podkrepibg.Campaigns.Application/Podkrepibg.Campaigns.Application.csproj]
#14 21.46
#14 21.46 Build FAILED.
#14 21.46
#14 21.46 /root/.nuget/packages/grpc.tools/2.37.0/build/_protobuf/Google.Protobuf.Tools.targets(280,5): error MSB6006: "/root/.nuget/packages/grpc.tools/2.37.0/tools/linux_arm64/protoc" exited with code 139. [/module-campaigns/src/Podkrepibg.Campaigns.Application/Podkrepibg.Campaigns.Application.csproj]
#14 21.46     0 Warning(s)
#14 21.46     1 Error(s)
#14 21.46
#14 21.46 Time Elapsed 00:00:20.98
------
executor failed running [/bin/sh -c dotnet build "Podkrepibg.Campaigns.csproj" -c Release -o /app/build]: exit code: 1
ERROR: Service 'campaigns' failed to build : Build failed
```
</details>

--- a/backend/module-campaigns/Dockerfile
+++ b/backend/module-campaigns/Dockerfile
@@ -6,6 +6,13 @@ EXPOSE 5001
 
 FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
 
+RUN apt-get update \
+ && DEBIAN_FRONTEND=noninteractive \
+    apt-get install --no-install-recommends --assume-yes \
+      protobuf-compiler-grpc
+
+ENV PROTOBUF_PROTOC=/usr/bin/protoc
+
 COPY module-campaigns/src/ module-campaigns/src/
 COPY protos/ protos/
 

